### PR TITLE
master

### PR DESCRIPTION
**Pull request by gvarao**
Added README.md for master

### CODE REVIEW

**Comment by ayushgithub** [comment]
Jenkins please retry a build

**Comment by gvarao** [comment]
Jenkins please retry a build

File: juniper_official/README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-Rules by juniper employees go here
+# Healthbot default KPIs
+#
+ 
+Contains readily consumable healthbot playbooks and rules which are specific to chassis, linecard, system, interface and protocols
+key performance indicators(KPIs). KPI rules collects the statistics from network devices then analyzes the data and act.
```
Comment (ConstrainedRandomVar) [inline juniper_official/README.md]: Change the "KPI rules.." sentence to:
"KPI rules collect the statistics from network devices, then analyze the data and act."

File: juniper_official/README.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+
+KPI playbook is set of rules, each rule is defined with set of KPIs.
```
Comment (ConstrainedRandomVar) [inline juniper_official/README.md]: Rephrase to "The KPI playbook is a set of rules, where each rule is defined with a set of KPIs."

File: juniper_official/README.md [hunk 3]
```diff
@@ -1 +1,16 @@
...
+
+Playbooks for chassis, linecard, system, interface and protocols(bgp, lldp, rib) are available in respective folders.
+
+Rules are defined with default variable values which can be changed while deploying playbook.
```
Comment (ConstrainedRandomVar) [inline juniper_official/README.md]: Change to:
Rules are defined with default variable values, which can be changed while deploying the playbook.

File: juniper_official/README.md [hunk 4]
```diff
@@ -1 +1,16 @@
...
+
+
+## Usage
+
+Apply the playbook to device-group under playbooks section using healthbot GUI.
```
Comment (ConstrainedRandomVar) [inline juniper_official/README.md]: Please change all instances of healthbot to HealthBot.
Also change the sentence to:
Apply the playbook to device-group under the playbooks section using the HealthBot GUI.

File: juniper_official/README.md (hunk 1 above)
Comment (gvarao) [reply juniper_official/README.md]: Corrected and committed.

File: juniper_official/README.md (hunk 2 above)
Comment (gvarao) [reply juniper_official/README.md]: corrected and commited

File: juniper_official/README.md (hunk 3 above)
Comment (gvarao) [reply juniper_official/README.md]: corrected and committed changes

File: juniper_official/README.md (hunk 4 above)
Comment (gvarao) [reply juniper_official/README.md]: Addressed

**Pull request merged**